core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -30,6 +31,8 @@ class SearchService(
 ) extends ClassfileIndexer
     with ClassfileListener
     with SLF4JLogging {
+
+  private val QUERY_TIMEOUT = 30 seconds
```

Comment:
We can leave these ones as they are all QUERY related, I thought perhaps they were also for the writes. Although I suspect we're not expecting an exception in the downstream usecase, which might result in blowing up the actors, but that's a different problem.